src/pages/VenuePage/VenuePage.tsx

```diff
@@ -84,11 +178,7 @@ export const VenuePage: React.FC = () => {
   const venueName = venue?.name ?? "";
   const venueTemplate = venue?.template;
 
-  const event = currentEvent?.[0];
-
-  const { isRelatedVenuesLoading } = useRelatedVenues({
-    currentVenueId: venueId,
-  });
```

Comment:
Why was this removed? (and/or what was it used for in the first place?

Reply:
Got added as part of a quick and dirty merge in GitHub UI that broke the code but allowed @mike-lvov to fix the rebase issue of https://github.com/sparkletown/sparkle/pull/1620 being behind staging.

Removed afterwards in 7489e0216306ef845d2e58085515fe0df99b1c27